app/models/allocation.rb

```diff
@@ -1,4 +1,6 @@
 class Allocation < ApplicationRecord
+  CURRENT_CYCLE_YEAR = "2020".freeze
```

Comment:
is it worth having this as a setting? eg `ALLOCATION_CYCLE_YEAR` so we know its different from the usual current/next cycle?

Reply:
Maybe near here https://github.com/DFE-Digital/teacher-training-api/blob/master/config/settings.yml#L6? so its all controlled from one place

Reply:
I've made 2 changes:
- Firstly I've renamed the constant so there's no confusion between it and the recruitment cycle year
- This is now configurable via the settings stuff